holidays/countries/brazil.py

```diff
@@ -69,77 +103,83 @@ def _populate_public_holidays(self):
         if self._year <= 1889:
             return None
 
-        # New Year's Day.
-        self._add_new_years_day("Confraternização Universal")
+        # Universal Fraternization Day.
+        self._add_new_years_day(tr("Confraternização Universal"))
 
         if 1892 <= self._year <= 1930:
             # Republic Constitution Day.
-            self._add_holiday_feb_24("Constituição da Republica")
+            self._add_holiday_feb_24(tr("Constituição da Republica"))
 
         # Good Friday.
-        self._add_good_friday("Sexta-feira Santa")
+        self._add_good_friday(tr("Sexta-feira Santa"))
 
         if self._year not in {1931, 1932}:
             # Tiradentes' Day.
-            self._add_holiday_apr_21("Tiradentes")
+            self._add_holiday_apr_21(tr("Tiradentes"))
 
         if self._year >= 1925:
-            # Labor Day.
-            self._add_labor_day("Dia do Trabalhador")
+            # Worker's Day.
```

Comment:
In fact, "Worker's Day" is indeed the correct translation. "Workers' Day" would be correct in case of "Dia dos Trabalhadores", where "trabalhadores" is the plural form of "worker" in portuguese.